bin/shpec

```diff
@@ -1,41 +1,51 @@
-#!/usr/bin/env bash
-VERSION=0.1.2
-examples=0
-failures=0
-test_indent=0
-red="\033[0;31m"
-green="\033[0;32m"
-norm="\033[0m"
+#
+: Use one of: bash, dash, or ksh93
```

Comment:
This is simply a comment, correct? Is there a reason it begins with a `:` and not a `#`?